packages/fxa-settings/src/components/Settings/PageMfaGuardWithGqlTest/index.tsx

```diff
@@ -0,0 +1,198 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useState, useEffect, useSyncExternalStore } from 'react';
+import {
+  JwtNotFoundError,
+  JwtTokenCache,
+  sessionToken as getSessionToken,
+} from '../../../lib/cache';
+
+import { MfaGuard } from '../MfaGuard';
+import { RouteComponentProps } from '@reach/router';
+import { useErrorHandler } from 'react-error-boundary';
+import { ApolloError, gql, useMutation } from '@apollo/client';
+
+export const PageMfaGuardTestWithGql = (props: RouteComponentProps) => {
+  return (
+    <MfaGuard requiredScope="test">
```

Comment:
<img width="1596" height="833" alt="image" src="https://github.com/user-attachments/assets/ca7dc161-063b-4f8f-905d-736991cd5e9b" />

I can't seem to get around this whether I input a valid code or an invalid code 🤔 But it seems to work okay for the auth-client version.